Server/DB/Scripts/rooms/addGuessToRoom.go

```diff
@@ -0,0 +1,51 @@
+package roomsScripts
+
+import (
+	"context"
+	"errors"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"ishtaloo.io/DB/Collections"
+	entities "ishtaloo.io/Entities"
+	wordUtils "ishtaloo.io/Utils/word"
+)
+
+func AddGuessToRoom(ctx context.Context, roomId string, guess rune) error {
+	if guess < 65 || guess > 90 {
+		return errors.New("guess is not a capital English letter")
+	}
+
+	roomsCollection := Collections.RoomsGetCollection()
+	id, _ := primitive.ObjectIDFromHex(roomId)
+	var room entities.Room
+	if err := roomsCollection.FindOne(ctx, bson.M{"_id": id}).Decode(&room); err != nil {
+		return err
+	}
+
+	takenGuesses := append(room.Guesses, wordUtils.VisibleRunes()...)
+	for _, g := range takenGuesses {
+		if g == guess {
+			return errors.New("guess already exists")
+		}
+	}
+	room.Guesses = append(room.Guesses, guess)
```

Comment:
taken guesses is referencing a user's guess? because I think we've  setteled that 2 users can choose the same guess

Reply:
But this guess is for the final guess, now?